### PR TITLE
Init version of about page

### PR DESCRIPTION
**Pull request by kokosing**
Init version of about page

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+
+## Who is going to write here?
+
+Mostly me. However, I will try to encourage others to write something also:
+
+ - colleagues from team. Presto is a huge project, it is impossible to be an expert of every part of it. In areas where I do not feel comfortable enough, I will try to ask an expert to give me a hand.
+
+ - Presto users from 3rd party companies. I will try to collect success stories from actual users.
+
+ - You? This blog is open-sourced. You can it source code [here](https://github.com/prestodb-rocks/site). Please do not hesitate to post an issue if you want me to write about something specific or post a pull request if you have something Presto related you want to share with the community. Any of these are very welcomed.
```
Comment (kokosing) [inline about.md]: add an email option

File: about.md [hunk 2]
```diff
@@ -1,9 +1,60 @@
...
+title: About site
 permalink: /about/
+author_profile: true
 ---
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
+## What is the aim of this blog?
 
-cd
+The main reason behind creating this blog is to make Presto to be more visible. So far it seems that this a first blog which is mainly devoted to this project.
```
Comment (nat925) [inline about.md]: Presto to be more visible -> bez "to"
a first -> the first

File: about.md [hunk 3]
```diff
@@ -1,9 +1,60 @@
...
+
+Besides I wanted to share the knowledge about Presto internals. About things are happening inside when run the query. From my experience I know that understanding how given tool works under the cover helps me a lot to use that tool. To avoid mistakes, to overcome obstacles, to achieve more.
```
Comment (nat925) [inline about.md]: things are -> things that are ... while running the query 
how given -> how the given

File: about.md [hunk 4]
```diff
@@ -1,9 +1,60 @@
...
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
+## What is the aim of this blog?
 
-cd
+The main reason behind creating this blog is to make Presto to be more visible. So far it seems that this a first blog which is mainly devoted to this project.
+
+Besides I wanted to share the knowledge about Presto internals. About things are happening inside when run the query. From my experience I know that understanding how given tool works under the cover helps me a lot to use that tool. To avoid mistakes, to overcome obstacles, to achieve more. 
+
+## What you will find here?
```
Comment (nat925) [inline about.md]: you will -> will you

File: about.md [hunk 5]
```diff
@@ -1,9 +1,60 @@
...
+
+I want to post here information about several different topics:
+ 
+ - features
+
+   About features of Presto. Not only how to use them and what behaviour expect, but also share the how it was implemented. I hope this will give you a notion when to use.
```
Comment (nat925) [inline about.md]: behaviour expect ->behaviour  to expect
the how -> how
when to use -> when to use what

File: about.md [hunk 6]
```diff
@@ -1,9 +1,60 @@
...
+
+ - tutorials
+
+   I hope you will find here also information like: how to create and install custom connector, configure kerberos, tune the query. Things that might be documented, but for sure do not have step-by-step explanation.
```
Comment (nat925) [inline about.md]: you will find here also -> you will also find here
custom connector -> a custom connector

File: about.md [hunk 7]
```diff
@@ -1,9 +1,60 @@
...
+
+   About features of Presto. Not only how to use them and what behaviour expect, but also share the how it was implemented. I hope this will give you a notion when to use.
+
+ - tutorials
+
+   I hope you will find here also information like: how to create and install custom connector, configure kerberos, tune the query. Things that might be documented, but for sure do not have step-by-step explanation. 
+
+ - success stories
+
+   I will try to find companies and user who successfully run Presto in production. It would be great if they share why they choose Presto, what do are doing with it, where it was great and where it caused pain, what setup do they have etc.?
```
Comment (nat925) [inline about.md]: user -> users

Comment (nat925) [inline about.md]: (chyba że to specjalnie)

File: about.md [hunk 8]
```diff
@@ -1,9 +1,60 @@
...
+
+ - ideas
+
+   Engineers and user often live in different worlds. Sometimes engineers get wrong notion about what users need. I would like to overcome that barrier here. I will write about ideas of Presto enhancements, which are not yet implemented (or even not yet planned) to get the early feedback to get know if it is a feature you are waiting for or it is just a waste of time.
```
Comment (nat925) [inline about.md]: user -> users
get wrong -> get a wrong
to get the early feedback to get know -> in order to learn / to get the early feedback if it is ...

File: about.md [hunk 9]
```diff
@@ -1,9 +1,60 @@
...
+
+ - numbers
+
+   Hopefully, benchmarks results will also find a place here. Either these conducted by Teradata or any other 3rd party company. I will try to add appropriate comment which could explain the reason why numbers look as they look.
```
Comment (nat925) [inline about.md]: appropriate -> an appropriate

File: about.md [hunk 10]
```diff
@@ -1,9 +1,60 @@
...
+
+## Is anything else than text can be found here?
```
Comment (nat925) [inline about.md]: Is anything else than text can be found here? -> Is there anything else other than a text to be found here? / Can anything other than a text be found here?

File: about.md [hunk 11]
```diff
@@ -1,9 +1,60 @@
...
+
+  Any utility script or program which I found useful in daily work with Presto and could be useful for other will be also published here.
```
Comment (nat925) [inline about.md]: and could be useful for other -> and which could be useful for others

File: about.md [hunk 12]
```diff
@@ -1,9 +1,60 @@
...
+
+## Who should read this?
+
+This blog should be read by Presto users. By people who want to broaden the knowledge about Presto, who are using it daily and want to streamline their work.
+
+## Who is going to write here?
+
+Mostly me. However, I will try to encourage others to write something also:
+
+ - colleagues from team. Presto is a huge project, it is impossible to be an expert of every part of it. In areas where I do not feel comfortable enough, I will try to ask an expert to give me a hand.
```
Comment (nat925) [inline about.md]: team -> my team/the team
it is -> so it is
expert of -> expert in

File: about.md [hunk 13]
```diff
@@ -1,9 +1,60 @@
...
+
+ - Presto users from 3rd party companies. I will try to collect success stories from actual users.
+
+ - You? This blog is open-sourced. You can it source code [here](https://github.com/prestodb-rocks/site). Please do not hesitate to post an issue if you want me to write about something specific or post a pull request if you have something Presto related you want to share with the community. Any of these are very welcomed.
```
Comment (nat925) [inline about.md]: can -> can find (bo chyba find zabrakło)
welcomed -> welcome

**Pull request merged**